app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -127,7 +127,8 @@ class PublicHealthHeader extends React.Component {
         <Modal.Body>
           {this.state.importType === 'saf' && (
             <div className="mb-3">
-              <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>
+              <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>{' '}
+              (Updated 12/2/2020)
```

Comment:
Can we perhaps get this date dynamically so people don't have to remember to update this date anytime they change the template?

I know if we get a `File` object we can get `lastModified` (see docs [here](https://developer.mozilla.org/en-US/docs/Web/API/File/lastModified))- so wondering if we can take advantage of that or if there's an even better way to do it. If not, not a big deal just was curious if we could be clever here.

Reply:
Resolving as the date actually needs to be the release date on prod anyway - so sticking with hardcoded solution for now.